run

```diff
@@ -462,8 +463,13 @@ build() {
          holoscan_sdk="-Dholoscan_ROOT=${sdk_path}"
          echo "Using Holscan SDK in ${sdk_path}"
          skipnext=1
+      elif [ "$arg" = "--with" ]; then
+         operator="${ARGS[i+1]}"
+         configure_args="${configure_args} -DHOLOHUB_BUILD_OPERATORS=${operator}"
+         echo "Building with operator ${operator}"
+         skipnext=1
       elif [ "$arg" = "--configure-args" ]; then
-         configure_args="${ARGS[i+1]}"
+         configure_args="${configure_args} ${ARGS[i+1]}"
```

Comment:
Looks like this should now allow multiple --configure-args options to be used in the command line